src/panels/lovelace/cards/hui-vertical-stack-card.ts

```diff
@@ -0,0 +1,102 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import computeCardSize from "../common/compute-card-size.js";
+import createCardElement from "../common/create-card-element.js";
+
+import { LovelaceCard, LovelaceConfig } from "../types.ts";
+
+interface Config extends LovelaceConfig {
+  cards: LovelaceConfig[];
+}
+
+class HuiVerticalStackCard extends LitElement
+  implements LovelaceCard {
+  protected hass?: HomeAssistant;
+  protected config: Config;
+  private elements: LovelaceCard[];
+
+  static get properties() {
+    return {
+      hass: {
+        observer: "_hassChanged",
+      },
+      config: {},
+    };
+  }
+
+  public getCardSize() {
+    let totalSize = 0;
+    this.elements.forEach((element) => {
+      totalSize += computeCardSize(element);
+    });
+    return totalSize;
+  }
+
+  public setConfig(config: Config) {
+    if (!config || !config.cards || !Array.isArray(config.cards)) {
+      throw new Error("Card config incorrect");
+    }
+
+    this.config = config;
+
+    if (this.shadowRoot) {
```

Comment:
This check isn't needed anymore. `_buildConfig()` can be moved into `setConfig`.